aiidalab/app.py

```diff
@@ -484,13 +486,13 @@ def _install_from_git_repository(self, git_url):
 
     def install(
         self,
-        version=None,
-        python_bin=None,
-        install_dependencies=True,
-        stdout=sys.stdout,
-        prereleases=False,
-        post_install_triggers=True,
-    ):
+        version: str | None = None,
+        python_bin: str | None = None,
+        install_dependencies: bool = True,
+        stdout: Any = sys.stdout,
```

Comment:
`sys.stdout` has type `Union[typing.TextIO, Any]` so for conciseness we'll simply use Any.